docs/source/contributing/developing.adoc

```diff
@@ -370,6 +371,31 @@ This information is often enough to find and debug the reason for a failure.
 However, for cases which require further investigation, the integration test also logs more detailed progress into a log file.
 This file is located at *_$GOPATH/github.com/minishift/minishift/out/test-results/integration_YYYY-MM-DD_HH:MM:SS.log_*.
 
+[[writing-features]]
+==== Writing features
+
+Minishift integration testsuite offers comprehensive set of implemented Gherkin steps which allows for comfortable writing of feature files.
+All available Gherkin steps are defined in file link:https://github.com/minishift/minishift/blob/master/test/integration/testsuite/testsuite.go#L75-L259[/test/integration/testsuite.go].
+If you are searching for particular step functionality, you can use following topic shortcuts:
+
+* link:https://github.com/minishift/minishift/blob/master/test/integration/testsuite/testsuite.go#L75-L81[Minishift status and profile verification]
```

Comment:
Using explicit line numbers for all of these links feels fragile to me. This list will have to be maintained if anything in **_testsuite.go_** changes.